flloat/ltlf.py

```diff
@@ -544,6 +544,37 @@ def _delta(self, i: PropositionalInterpretation, epsilon=False):
         return self.to_nnf()._delta(i, epsilon=epsilon)
 
 
+class LTLfLast(LTLfFormula):
+    """Class for the LTLf Last formula."""
+
+    def _delta(self, i: PropositionalInterpretation, epsilon=False):
+        """Apply the delta function."""
+        return self.to_nnf()._delta(i, epsilon=epsilon)
+
+    def to_nnf(self) -> LTLfFormula:
+        """Transform to NNF."""
+        return LTLfAnd([LTLfWeakNext(LTLfFalse()), LTLfNot(LTLfEnd())]).to_nnf()
+
+    def negate(self) -> LTLfFormula:
+        """Negate the formula."""
+        return LTLfNext(LTLfTrue())
+
+    def truth(self, i: FiniteTrace, pos: int = 0):
+        """Evaluate the formula."""
+        return self.to_nnf().truth(i, pos)
+
+    def find_labels(self) -> Set[AtomSymbol]:
+        """Find the labels."""
+        return self.to_nnf().find_labels()
```

Comment:
Maybe its better to return empty set direclty?

Reply:
yes!